packages/styles/scss/patterns/leadspace/_leadspace.scss

```diff
@@ -5,11 +5,141 @@
 // LICENSE file in the root directory of this source tree.
 //
 
-@import '../../globals/imports';
-
 @mixin leadspace {
   .#{$prefix}--leadspace {
-    background-color: transparent;
+    position: relative;
+    padding-top: carbon--rem(370px) / carbon--mini-units(40) * 100%;
+    height: 0;
+    background: $ui-background;
+
+    &__image {
+      object-fit: cover;
+      width: 100%;
+      height: 100%;
+    }
+
+    &__container {
+      position: absolute;
+      top: 0;
+      height: 100%;
+      width: 100%;
+    }
+
+    &__overlay {
+      @include carbon--make-container;
+
+      padding-top: $layout-03;
+      padding-bottom: $layout-03;
+      position: absolute;
+      top: 0;
+      height: 100%;
+      width: 100%;
+      max-width: none;
+      background-image: linear-gradient(
+        to bottom,
+        rgba($ui-background, 1),
+        transparent
+      );
+    }
+
+    &__content {
+      position: absolute;
+      bottom: $layout-03;
+    }
+
+    .#{$prefix}--leadspace__title {
+      @include carbon--type-style(display-01, true);
+
+      color: $text-01;
+    }
+
+    .#{$prefix}--leadspace__desc {
+      @include carbon--type-style(expressive-heading-03, true);
+
+      color: $text-01;
+    }
+
+    &__ctas {
+      display: flex;
+      flex-direction: column-reverse;
+      margin-top: $layout-01;
+    }
+
+    .#{$prefix}--btn {
+      width: 100%;
+      margin-top: $layout-01;
+    }
+  }
+
+  .#{$prefix}--leadspace--productive .#{$prefix}--leadspace__title {
+    @include carbon--type-style(expressive-heading-06, true);
+  }
+
+  @include carbon--breakpoint(md) {
+    .#{$prefix}--leadspace {
+      padding-top: carbon--mini-units(50) / carbon--mini-units(84) * 100%;
+
+      &__overlay {
+        background-image: linear-gradient(
+          to right,
+          rgba($ui-background, 1),
+          transparent 50%
+        );
+      }
+
+      &__content {
+        width: 100%;
+      }
+
+      .#{$prefix}--leadspace__title {
+        @include carbon--make-row;
+        @include carbon--make-col-ready;
+        @include carbon--make-col(7, 8);
+      }
+
+      .#{$prefix}--leadspace__desc {
+        width: 50%;
+      }
+
+      &__ctas {
+        flex-direction: row-reverse;
+        float: left;
+        margin-top: $layout-03;
+      }
+
+      .#{$prefix}--btn {
+        width: auto;
+        margin-top: 0;
+        margin-right: $layout-01;
+      }
+    }
+  }
+
+  @include carbon--breakpoint(lg) {
+    .#{$prefix}--leadspace {
+      padding-top: carbon--mini-units(60) / carbon--mini-units(132) * 100%;
+
+      .#{$prefix}--leadspace__title {
+        @include carbon--make-col(8, 16);
+      }
+
+      .#{$prefix}--leadspace__desc {
+        @include carbon--make-row;
+        @include carbon--make-col-ready;
+        @include carbon--make-col(4, 16);
```

Comment:
We're having a slightly different alignment issue down here.

![image](https://user-images.githubusercontent.com/3793636/65165425-5c03fb80-da04-11e9-9ea3-27111ff800b9.png)

Reply:
With that said for some reason when I go to add 4 more columns it just stacks them.

![image](https://user-images.githubusercontent.com/3793636/65165644-d16fcc00-da04-11e9-80b8-b92e1d819a43.png)

Reply:
how are you adding the 4 columns? using the mixins or the grid classes?

Reply:
I removed the `absolute` positioning from the `content` div so it looks a bit different...slightly off still

<img width="484" alt="Screen Shot 2019-09-19 at 11 00 10 AM" src="https://user-images.githubusercontent.com/54281166/65255959-c3857e00-dacc-11e9-9435-09648c79e4d2.png">